src/System.Threading/tests/System.Threading.Tests.builds

```diff
@@ -9,7 +9,7 @@
     </Project>
     <Project Include="System.Threading.Tests.csproj">
       <OSGroup>Windows_NT</OSGroup>
-      <TestTFMs>net46</TestTFMs>
+      <TestTFMs>net463</TestTFMs>
```

Comment:
It will compile the test library using the netstandard1.3 surface area, which is the one supported by net46. If you don't add it, you would be using this project's default to build (in this case netstandard1.7) which might break at runtime as netstandard1.7 has API that wasn't present on net46